debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/StreamingAdapter.java

```diff
@@ -21,14 +21,18 @@
  */
 public interface StreamingAdapter {
 
+    void configure(OracleConnectorConfig connectorConfig, OracleConnection connection);
+
     String getType();
 
     HistoryRecordComparator getHistoryRecordComparator();
 
-    OffsetContext.Loader getOffsetContextLoader(OracleConnectorConfig connectorConfig);
+    OffsetContext.Loader getOffsetContextLoader();
 
-    StreamingChangeEventSource getSource(OracleConnectorConfig connectorConfig, OffsetContext offsetContext,
-                                         OracleConnection connection, EventDispatcher<TableId> dispatcher, ErrorHandler errorHandler, Clock clock,
-                                         OracleDatabaseSchema schema, OracleTaskContext taskContext, Configuration jdbcConfig,
+    StreamingChangeEventSource getSource(OffsetContext offsetContext, EventDispatcher<TableId> dispatcher,
+                                         ErrorHandler errorHandler, Clock clock, OracleDatabaseSchema schema,
+                                         OracleTaskContext taskContext, Configuration jdbcConfig,
                                          OracleStreamingChangeEventSourceMetrics streamingMetrics);
+
+    boolean getTablenameCaseInsensitivity();
```

Comment:
Let's add a JavaDoc comment. And also return an enum rather than boolean, as it's more descriptive.